src/test/e2e/13_find_images_test.go

```diff
@@ -0,0 +1,53 @@
+package test
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFindImages(t *testing.T) {
+	t.Log("E2E: Find Images")
+
+	t.Run("zarf test find images success", func(t *testing.T) {
+		t.Log("E2E: Test Find Images")
+
+		testPackagePath := filepath.Join("examples", "dos-games")
+		expectedOutput, err := os.ReadFile("src/test/packages/13-find-images/dos-games-find-images-expected.txt")
+		require.NoError(t, err)
+
+		stdout, _, err := e2e.Zarf("dev", "find-images", testPackagePath)
+		require.NoError(t, err)
+		require.Contains(t, stdout, string(expectedOutput))
+	})
+
+	t.Run("zarf test find images --why  w/ helm chart success", func(t *testing.T) {
+		t.Log("E2E: Test Find Images against a helm chart with why flag")
+
+		testPackagePath := filepath.Join("examples", "helm-charts")
+		expectedOutput, err := os.ReadFile("src/test/packages/13-find-images/helm-charts-find-images-why-expected.txt")
+		require.NoError(t, err)
+
+		stdout, _, err := e2e.Zarf("dev", "find-images", testPackagePath, "--why", "curlimages/curl:7.69.0")
+		require.NoError(t, err)
+		match, err := regexp.MatchString(string(expectedOutput), stdout)
+		require.NoError(t, err)
+		require.True(t, match)
+	})
```

Comment:
This test case does not validate the expected output accurately because of the use of `regexp.MatchString`. The pod that uses the curl image in the podinfo chart has a dynamically-generated pod name, leading to non-deterministic output, which makes this example unsuitable for a good test case.

If you run the the command with the --why flag locally for the curl image against examples/helm-charts, you will see that the output correctly only has 3 resources printed to stdout, but the test txt file has 6.

https://github.com/stefanprodan/podinfo/blob/47090ad9e14e60f33c3d562b5ca626742c8d1309/charts/podinfo/templates/tests/service.yaml#L4

Recommend using the `examples/wordpress` helm chart as a test case instead so that we can have deterministic output and more accurately assert against it.

```suggestion
	t.Run("zarf test find images --why  w/ helm chart success", func(t *testing.T) {
		t.Log("E2E: Test Find Images against a helm chart with why flag")

		testPackagePath := filepath.Join("examples", "wordpress")
		expectedOutput, err := os.ReadFile("src/test/packages/13-find-images/helm-charts-find-images-why-expected.txt")
		require.NoError(t, err)

		stdout, _, err := e2e.Zarf("dev", "find-images", testPackagePath, "--why", "docker.io/bitnami/apache-exporter:0.13.3-debian-11-r2")
		require.NoError(t, err)
		require.Contains(t, stdout, string(expectedOutput))
	})
```